finality-aleph/src/network.rs

```diff
@@ -312,7 +304,7 @@ struct SessionData<D> {
 
 #[derive(Clone, Encode, Decode)]
 enum SessionCommand<D: Clone + Encode + Decode> {
-    Meta(MetaMessage, Recipient<PeerId>),
+    Meta(Box<MetaMessage>, Recipient<PeerId>),
```

Comment:
why box is now required?

Reply:
clippy, becasue variant is large, wanted it on the heap